west.yml

```
@@ -146,7 +146,7 @@ manifest:
       - homekit
     - name: find-my
       repo-path: sdk-find-my
-      revision: v2.0.0-rc2
+      revision: 5f80737fd923e200cb93e9d77bd4d9fbefa1e505
```

Please redirect the manifest file to your Find My PR so that the Find My CI can check the new change

```suggestion
      revision: pull/90/head
```

The Find My CI has no access to the commit hash that does not belong to the upstream repo (your commit is in your private fork). That's why it fails here:

https://jenkins-ncs.nordicsemi.no/blue/organizations/jenkins/latest%2Fsub%2Ftest-fw-nrfconnect-nrf/detail/master/23906/pipeline

---

fixed